src/datasources/alarms/components/AlarmsVariableQueryEditor.test.tsx

```diff
@@ -0,0 +1,104 @@
+import { screen, waitFor } from '@testing-library/react';
+import { Workspace } from 'core/types';
+import { AlarmsDataSource } from '../AlarmsDataSource';
+import { AlarmsVariableQuery } from '../types/types';
+import { setupRenderer } from 'test/fixtures';
+import { AlarmsVariableQueryEditor } from './AlarmsVariableQueryEditor';
+
+const fakeWorkspaces: Workspace[] = [
+    {
+        id: 'ws1',
+        name: 'Test Workspace 1',
+        default: false,
+        enabled: true
+    },
+    {
+        id: 'ws2', 
+        name: 'Test Workspace 2',
+        default: true,
+        enabled: true
+    }
+];
+
+class FakeAlarmsDataSource extends AlarmsDataSource {
+    constructor() {
+        super({} as any);
+        
+        (this as any)._listAlarmsDataSource = {
+            async loadWorkspaces(): Promise<Map<string, Workspace>> {
+                const workspacesMap = new Map<string, Workspace>();
+                fakeWorkspaces.forEach(ws => workspacesMap.set(ws.id, ws));
+                return Promise.resolve(workspacesMap);
+            },
+            
+            globalVariableOptions() {
+                return [
+                    { label: 'Workspace', value: 'workspace' },
+                    { label: 'Alarm ID', value: 'alarmId' }
+                ];
+            },
+
+            get errorTitle() {
+                return undefined as string | undefined;
+            },
+
+            get errorDescription() {
+                return undefined as string | undefined;
+            }
+        };
+    }
+}
+
+const render = setupRenderer(AlarmsVariableQueryEditor, FakeAlarmsDataSource, () => {});
+
+describe('AlarmsVariableQueryEditor', () => {
+    it('renders the query builder', async () => {
```

Comment:
```suggestion
    it('should renders the query builder', async () => {
```

Please apply for all similar cases